android/src/main/kotlin/com/os/operando/advertisingid/AdvertisingIdPlugin.kt

```diff
@@ -14,7 +14,7 @@ import io.flutter.plugin.common.PluginRegistry.Registrar
 import kotlin.concurrent.thread
 
 class AdvertisingIdPlugin() : FlutterPlugin, ActivityAware, MethodCallHandler {
-    private lateinit var activity: Activity
+    public lateinit var activity: Activity
```

Comment:
@jocelyngriselle 
Why did you change it to public?

Reply:
sorry no reason it's a typo